### ISSUE TITLE
memory corruption in seterr if table is too large?

### ISSUE THREAD

**Issue by coldfix** [coldfix/madx-dummy#178]
**Originally by _yngve.inntjore.levinsen@cern.ch_ on 2012-10-01 18:16:22 +0200**

___
date: 2012.10.01
I had an error table in a file I tried to read into Mad-X but I got an error
```
*** glibc detected *** madx_dev: malloc(): memory corruption: 0x00000000202af6a0 ***
```
I figured out that by reducing the number of errors in the table, the script suddenly worked. Hence I am wondering if this might be some hard-coded limit to the list of errors which can be assigned or similar. Attaching files as example.

**Comment by coldfix** [comment]
**Originally by _yngve.inntjore.levinsen@cern.ch_ on 2012-10-01 18:16:53 +0200**

___
**Attachment added:** [`bug.madx`](https://raw.githubusercontent.com/coldfix/madx-tickets/master/attachments/178/bug.madx) (0.6 KiB)
madx script

**Comment by coldfix** [comment]
**Originally by _yngve.inntjore.levinsen@cern.ch_ on 2012-10-01 18:17:07 +0200**

___
**Attachment added:** [`error.small.dat`](https://raw.githubusercontent.com/coldfix/madx-tickets/master/attachments/178/error.small.dat) (79.7 KiB)
error table

**Comment by coldfix** [comment]
**Originally by _ldeniau_ on 2012-12-04 17:41:12 +0100**

___
date: 2012.12.04, from laurent.deniau
The problem is corrected (starting from 5.00.19). The function error_seterr was accessing incorrectly arrays.
